src/arviz_plots/plots/parallel_plot.py

```diff
@@ -0,0 +1,229 @@
+            "line",
+            "xticks",
+        ],
+        Sequence[str],
+    ] = None,
+    visuals: Mapping[
+        Literal[
+            "line",
+            "divergence",
+            "xticks",
+        ],
+        Mapping[str, Any] | Literal[False],
+    ] = None,
+    **pc_kwargs,
+):
+    """Plot parallel coordinates plot showing posterior points with and without divergences.
+
+    Parameters
+    ----------
+    dt : DataTree
+        Input data
+    var_names: str or list of str, optional
+        One or more variables to be plotted.
+        Prefix the variables by ~ when you want to exclude them from the plot.
+    filter_vars: {None, “like”, “regex”}, default None
+        If None (default), interpret var_names as the real variables names.
+        If “like”, interpret var_names as substrings of the real variables names.
+        If “regex”, interpret var_names as regular expressions on the real variables names.
+    group : str, default "posterior"
+        Group to use for plotting. Defaults to "posterior".
+    coords : mapping, optional
+        Coordinates to use for plotting.
+    sample_dims : iterable, optional
+        Dimensions to reduce unless mapped to an aesthetic.
+        Defaults to ``rcParams["data.sample_dims"]``
+    plot_collection : PlotCollection, optional
+    backend : {"matplotlib", "bokeh", "plotly", "none"}, optional
+        Plotting backend to use. Defaults to ``rcParams["plot.backend"]``
+    labeller : labeller, optional
+    aes_by_visuals : mapping, optional
+        Mapping of visuals to aesthetics that should use their mapping in `plot_collection`
+        when plotted. Valid keys are the same as for `visuals`.
+        By default, there are no aesthetic mappings at all
+    visuals : mapping of {str : mapping or False}, optional
+        Valid keys are:
+
+        * line -> passed to :func:`~.visuals.multiple_lines`
+        * divergence -> not passed anywhere, can only be set ``False`` to avoid divergence plotting.
+        * xticks -> passed to :func:`~.visuals.set_xticks`. Defaults to False.
+
+
+    **pc_kwargs
+        Passed to :meth:`arviz_plots.PlotCollection.wrap`
+
+
+    Returns
+    -------
+    PlotCollection
+
+    Examples
+    --------
+    Default plot_parallel
+
+    .. plot::
+        :context: close-figs
+
+        >>> from arviz_plots import plot_parallel, style
+        >>> style.use("arviz-variat")
+        >>> from arviz_base import load_arviz_data
+        >>> dt = load_arviz_data('centered_eight')
+        >>> plot_parallel(
+        >>>     dt,
+        >>>     var_names=["mu", "tau"],
+        >>> )
+
+    .. minigallery:: plot_parallel
+
+    """
+    if sample_dims is None:
+        sample_dims = rcParams["data.sample_dims"]
+    if isinstance(sample_dims, str):
+        sample_dims = [sample_dims]
+    if visuals is None:
+        visuals = {}
+    if pc_kwargs is None:
+        pc_kwargs = {}
+    else:
+        pc_kwargs = pc_kwargs.copy()
```

Comment:
I didn't realize this when we updated the call signature of the functions. By using `**pc_kwargs` is is always a dictionary and it is defined internally when calling the function, it can't be a user defined dictionary that they also have stored as a variable. We can get rid of the whole if else.

We still need to copy mutable values inside `pc_kwargs` like figure_kwargs or aes before modifying them though.